tokio-console/src/view/task.rs

```diff
@@ -341,3 +377,53 @@ fn dur(styles: &view::Styles, dur: std::time::Duration) -> Span<'static> {
     // have to be given a string in order to do layout stuff?
     styles.time_units(format!("{:.prec$?}", dur, prec = DUR_PRECISION))
 }
+
+fn display<W: fmt::Write>(
```

Comment:
can we maybe give this a more descriptive name? this is specifically for displaying stack elements, right? so maybe something that indicates that?

alternatively, we could make it an inner function of the `make_stack_widget` function, i suppose.